src/cli/hook-utils/manifest.js

```diff
@@ -2,12 +2,18 @@ const fs = require('fs');
 const path = require('path');
 const merge = require('deepmerge');
 
-// helper array merge function
-function unionMerge(array1, array2) {
+/** 
+ * Union merge of arrays
+ */
+export function unionMerge(array1, array2) {
```

Comment:
@srajiang it seems like this `export` keyword here threw an error for me when running `hermes manifest` while testing. i think since it's getting exported via `module.exports` it should be good to remove this `export` keyword!

Reply:
Good catch @ashleymcm 🙇‍♀️ - I've updated to remove the unnecessary export